packages/web-components/fast-element/src/observation/splice-strategies.ts

```diff
@@ -1,4 +1,10 @@
 import { emptyArray } from "../platform.js";
+import {
+    ArrayObserver,
+    Splice,
+    SpliceStrategy,
+    SpliceStrategySupport,
+} from "./arrays.js";
 
 const enum Edit {
```

Comment:
We usually favor types instead of enums.

Reply:
This one was purely internal to make the code more readable and gets compiled out entirely due to the `const enum` setting.